sklearn/model_selection/tests/test_split.py

```diff
@@ -783,6 +783,17 @@ def train_test_split_pandas():
         assert_true(isinstance(X_test, InputFeatureType))
 
 
+def train_test_split_sparse():
+    # check that train_test_split converts scipy sparse matrices to csr
```

Comment:
Well, CSR and LIL (and maybe BSR) are certainly easiest to split by row, which is what CV splitters do.